unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarAssets/WearableItem.cs

```diff
@@ -48,6 +48,22 @@ public class Data
     public int issuedId;
 
     private readonly Dictionary<string, string> cachedI18n = new Dictionary<string, string>();
+    private readonly Dictionary<string, ContentProvider> cachedContentProviers =
+        new Dictionary<string, ContentProvider>();
+
+    private readonly string[] skinImplicitCategories =
+    {
+        // TODO: this is not supported by the new avatar renderer system and can cause problems
+        // WearableLiterals.Categories.EYES,
+        // WearableLiterals.Categories.MOUTH,
+        // WearableLiterals.Categories.EYEBROWS,
```

Comment:
```suggestion

```

Reply:
these categories can be added with the new avatar renderer support